src/services/storage.ts

```diff
@@ -70,12 +73,12 @@ export class Storage extends Service {
      *
      * @param {string} bucketId
      * @param {string} fileId
-     * @param {{name: string, type: string, size: number, uri: string}} file
+     * @param {any} file
      * @param {string[]} permissions
      * @throws {AppwriteException}
      * @returns {Promise}
     */
-    async createFile(bucketId: string, fileId: string, file: {name: string, type: string, size: number, uri: string}, permissions?: string[], onProgress = (progress: UploadProgress) => {}): Promise<Models.File> {
+    async createFile(bucketId: string, fileId: string, file: any, permissions?: string[], onProgress = (progress: UploadProgress) => {}): Promise<Models.File> {
```

Comment:
How come the file type was changed to `any`? The typing was kind of helpful to figure out what needed to be passed in here.